Translation of all lines into Russian.

dean-win opened the pull request.

GuideDev commented:
Awesome, thanks for contributing! I'll include this in the next release. Let me know if you'd like me to be credited as something other than dean-win :slightly_smiling_face: 

Also, let me know if you're on Discord so I can tag you as a translator there in case you want a heads up whenever new keys come out or something.

alex19EP commented:
hello @GuideDev I and sevapopov2
already started on Russian translation. we will rebace hour work on latest master and, send improvements.

@dean-win it would be nice if you could join discord and discuss translation improvements with us.

alex19EP commented:
@GuideDev This translation contains a large number of mistakes. If I were you, I would revert this translation before a new release.

GuideDev commented:
I've left it out of today's release while people review and submit improvements. I don't understand Russian at all unfortunately so please work it out between yourselves and submit a PR with any improvements you see fit. Just let me know once you're all happy with it :slightly_smiling_face:

dean-win commented:
@GuideDev I am on discord. dean-win#0711

dean-win commented:
@alex19EP Hello! What errors have been corrected in the translation? Below I will throw off the lines that should be checked before sending. We can contact you to discuss the translation and any errors you find.
In your translation, in addition to technical errors, there are also grammatical ones.
I am not writing to offend you.

Забрать 0 |4(награду,награды) 1 из 2.
Ежедневно 1 из 2. (While in the Russian translation of the game, these tasks are called "ежедневные", there may be several).
Quite a lot of questions about this translation.

"В колоде: 25 |4(карта,карт)."
"В руке противника. 6 |4(карта,карт)."
Противник взял 1 карта."
"Ваш Ядовитый скорпид разморожена."
"Здоровье 22."

I am ready to communicate and ready to discuss if you think that there are errors in my version of the translation. I don't rule it out and ready to fix it.

dean-win commented:
I was wrong about"ежедневно".
I thought about the cards, but wrote what came across. 😂
This is not originally translated.
I found some points in your translation that will sound better than mine.
Can contact and discuss.

dean-win commented:
@GuideDev Hello! I have updated the translation. Communicated with Russian players, made changes and fixed bugs. Also, I took some points from the translation of Seva Popov. Unfortunately, I was unable to contact them. @alex19EP does not appear on Discord.
